ArduCopter/Parameters.h

```diff
@@ -368,6 +368,12 @@ class Parameters {
 
         // 254,255: reserved
 
+        //
+        // Pilot_Velocity Extras - 260
+        //
+        k_param_pilot_velocity_z_max_up = 260,
+        k_param_pilot_velocity_z_max_dn = 261,
```

Comment:
This needs to be verified, but I think we can't use these numbers, that's why we already have another group of parameters.

Reply:
Hi @OXINARF, I must be misunderstanding something?  I'm still learning the codebase (and my c++ skills are a little rusty) so apologies if I'm making basic mistakes.  I gave the new ones a new block and gave them values not in the enum list?  

Thanks for the above git suggestions, I'll give it a go when I get home.